packages/node/src/lib/__tests__/get-message-id.test.ts

```diff
@@ -0,0 +1,21 @@
+import { createMessageId } from '../get-message-id'
+
+// https://gist.github.com/johnelliott/cf77003f72f889abbc3f32785fa3df8d
+const uuidv4Regex =
+  /^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i
+
+describe(createMessageId, () => {
+  it('creates a messageId that may look like "node-next-166820896773a9-c7404832-feea-4e70-ac7f-2dc94d96acca"', () => {
```

Comment:
I think the messageId in this example is incorrect - Date.now() and the first part of the uuid look smoosed.